index.js

```diff
@@ -9,8 +9,7 @@ module.exports = function (string) {
   // normalize git@ and https:git@ urls
   string = string.replace(/^git@/, 'https://')
   string = string.replace(/^https:git@/, 'https://')
-  string = string.replace('.com:', '.com/')
-
+  string = string.replace(/\.([A-Za-z]+)\:/, '.$1/');
```

Comment:
The pattern used for matching the TLD will need to be changed at a later time. Non-ascii characters are allowed, and used, for international domains.

Reply:
Good point